app/models/article.rb

```diff
@@ -4,4 +4,19 @@ class Article < ApplicationRecord
   has_many :likes, dependent: :destroy
   has_rich_text :content
   validates :title, presence: true, uniqueness: true, length: { in: 5..30 }
+  def total_likes
+    likes.count
+  end
+  def liking_users
```

Comment:
Add empty line before and after the method

Reply:
okay